bld010/frontend-palette-picker
# Fetch calls

#### What's this PR do?

This PR adds fetch calls for each of GET, PATCH, POST, and DELETE for both folders and palettes
It also creates a util directory(for apiCalls and cleaners) and a .env file for determining urls

#### Where should the reviewer start?

the util directory and both cleaners and apiCalls

#### How should this be manually tested?

open the react app and check the state of the app component and ensure that folders has been set to the folders and palettes that have been retrieved and sent back. 

#### Any background context you want to provide?

n/a


#### What are the relevant tickets?

[ticket](https://app.clubhouse.io/briannadelvalle/story/61/as-a-developer-i-want-to-fetch-all-palettes-and-folders-from-backend-and-save-them-in-app-so-i-can-display-them-for-the-user)


#### Screenshots (if appropriate)

n/a


#### Questions:

n/a
